src/libs/metadata.c

```diff
@@ -745,6 +745,7 @@ void gui_init(dt_lib_module_t *self)
     gtk_grid_attach(grid, swindow, 1, i, 1, 1);
     gtk_widget_set_hexpand(swindow, TRUE);
     d->swindow[i] = swindow;
+    gtk_scrolled_window_set_policy(GTK_SCROLLED_WINDOW(swindow), GTK_POLICY_AUTOMATIC, GTK_POLICY_AUTOMATIC);
     gtk_widget_set_size_request(d->swindow[i], -1, DT_PIXEL_APPLY_DPI(30));
 
     gtk_text_view_set_wrap_mode(GTK_TEXT_VIEW(textview), GTK_WRAP_WORD);
```

Comment:
Changing GTK_WRAP_WORD to GTK_WRAP_WORD_CHAR you fix also the case of very long word.
Sorry, 3 lines below